frontend/src/app/sessions/feedback/feedback.service.ts

```diff
@@ -4,38 +4,30 @@
  */
 import { Injectable } from '@angular/core';
 import { MatDialog } from '@angular/material/dialog';
-import { BehaviorSubject, Observable, tap } from 'rxjs';
+import { map } from 'rxjs';
 import { AuthenticationWrapperService } from 'src/app/services/auth/auth.service';
-import {
-  FeedbackConfigurationOutput,
-  FeedbackService as OpenAPIFeedbackService,
-  Session,
-} from '../../openapi';
+import { Session } from '../../openapi';
+import { UnifiedConfigWrapperService } from '../../services/unified-config-wrapper/unified-config-wrapper.service';
 import { FeedbackDialogComponent } from './feedback-dialog/feedback-dialog.component';
 
 @Injectable({
   providedIn: 'root',
 })
 export class FeedbackWrapperService {
   constructor(
-    private feedbackService: OpenAPIFeedbackService,
     public dialog: MatDialog,
     private authService: AuthenticationWrapperService,
+    public unifiedConfigWrapperService: UnifiedConfigWrapperService,
   ) {
-    this.loadFeedbackConfig().subscribe(() => this.triggerFeedbackPrompt());
+    this.unifiedConfigWrapperService.unifiedConfig$.subscribe(() => {
+      this.triggerFeedbackPrompt();
+    });
   }
 
-  private _feedbackConfig = new BehaviorSubject<
-    FeedbackConfigurationOutput | undefined
-  >(undefined);
-
-  public readonly feedbackConfig$ = this._feedbackConfig.asObservable();
-
-  loadFeedbackConfig(): Observable<FeedbackConfigurationOutput> {
-    return this.feedbackService
-      .getFeedbackConfiguration()
-      .pipe(tap((feedbackConf) => this._feedbackConfig.next(feedbackConf)));
-  }
+  public readonly feedbackConfig$ =
+    this.unifiedConfigWrapperService.unifiedConfig$.pipe(
+      map((unifiedConfig) => unifiedConfig?.feedback),
+    );
 
   triggerFeedbackPrompt(): void {
```

Comment:
The `triggerFeedbackPrompt` function is called directly after authentication is completed. At this time, the unified configuration is not loaded. Therefore, we should change it to use the Observable: 

```py
triggerFeedbackPrompt(): void {
    if (!this.authService.isLoggedIn()) return;
    this.feedbackConfig$.pipe(filter(Boolean), take(1)).subscribe((config) => {
      if (this.shouldShowIntervalPrompt(config)) {
        this.showDialog([], 'On interval');
        this.saveFeedbackPromptDate();
      }
    });
  }
```